weaviate/weaviate_client.go

```diff
@@ -180,11 +180,13 @@ func NewClient(config Config) (*Client, error) {
 }
 
 // New client from config
-// Every function represents one API group of weaviate and provides a set of functions and builders to interact with them.
-//
-// The client uses the original data models as provided by weaviate itself.
-// All these models are provided in the sub module "github.com/weaviate/weaviate/entities/models"
+// For backwards compatibility, this function will panic if the client cannot be created
```

Comment:
```suggestion
```
I think adding this line is unnecessary, because we won't be updating any code related to the panic